### PR TITLE
Add TLS&auth and subqueries to roadmap

### PR DESCRIPTION
**Pull request by juliusv**
Signed-off-by: Julius Volz <julius.volz@gmail.com>

### CODE REVIEW

**Comment by juliusv** [comment]
@SuperQ @brian-brazil

File: content/docs/introduction/faq.md [hunk 1]
```diff
@@ -221,6 +221,11 @@ second, will only run into precision issues after over 285 years.
 
 ### Why don't the Prometheus server components support TLS or authentication? Can I add those?
 
+Note: The Prometheus team has recently changed their stance on this and
+support for TLS and authentication in serving endpoints is now on the
+[project's roadmap](/docs/introduction/roadmap/#tls-and-authentication-in-http-serving-endpoints).
+As long as this is not implemented yet, the advice below still applies.
```
Comment (brian-brazil) [inline content/docs/introduction/faq.md]: Even when implemented, much of the below still applies.

Comment (RichiH) [reply content/docs/introduction/faq.md]: "This document will be updated once code changes have been made."?

File: content/docs/introduction/faq.md [hunk 2]
```diff
@@ -221,6 +221,11 @@ second, will only run into precision issues after over 285 years.
 
 ### Why don't the Prometheus server components support TLS or authentication? Can I add those?
 
+Note: The Prometheus team has recently changed their stance on this and
```
[the remark this answers is not in the document]
Comment (juliusv) [reply content/docs/introduction/faq.md]: Done.

File: content/docs/introduction/faq.md (hunk 1 above)
Comment (juliusv) [reply content/docs/introduction/faq.md]: Replaced with @RichiH's suggestion.

**Pull request merged**